projects/community-payback-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/model/TeamsResponse.kt

```diff
@@ -0,0 +1,17 @@
+package uk.gov.justice.digital.hmpps.model
+
+import uk.gov.justice.digital.hmpps.integrations.delius.Team
+
+data class TeamsResponse(
+    val teams: List<TeamDTO>
+)
+
+data class TeamDTO(
+    val code: String,
+    val description: String
+)
```

Comment:
Is it worth making this a generic `CodeDescription`-type class? I expect there'll be a few other response objects in this format from the other endpoints.